builtin/rebase.c

```diff
@@ -0,0 +1,54 @@
+/*
+ * "git rebase" builtin command
+ *
+ * Copyright (c) 2018 Pratik Karki
+ *
+ */
+
+#include "builtin.h"
+#include "run-command.h"
+#include "exec-cmd.h"
+#include "parse-options.h"
+/*
+static const char * const rebase_usage[] = {
+	NULL
+};
+
+static struct option rebase_options[] = {
+	OPT_END()
+};
+*/
+static int use_builtin_rebase(void) {
+	struct child_process cp = CHILD_PROCESS_INIT;
+	struct strbuf out = STRBUF_INIT;
+	int ret;
+
+	argv_array_pushl(&cp.args,
+			 "config", "--bool", "rebase.usebuiltin", NULL);
+	cp.git_cmd = 1;
+	if (capture_command(&cp, &out, 28))
```

Comment:
I wonder if `out`should be released before returning in case capture_command() failed.

Also I wonder why 28 is used.

Reply:
Oh no.... At first I was trying to options provided by rebase into the builtin. There were about 28 options provided in rebase. But, that was not the approach to do it and I failed to do so. I think I mistakenly left 28 there.